hip-clang.docker

```diff
@@ -61,5 +65,8 @@ RUN cget -p $PREFIX install RadeonOpenCompute/rocm-cmake@master
 RUN cget -p $PREFIX install pfultz2/rocm-recipes
 ADD min-requirements.txt /min-requirements.txt
 RUN CXXFLAGS='-isystem $PREFIX/include' cget -p $PREFIX install -f /min-requirements.txt
-
+RUN export HIPCC_LINK_FLAGS_APPEND='-O3 -parallel-jobs=4'
+RUN export HIPCC_COMPILE_FLAGS_APPEND='-O3 -Wno-format-nonliteral -parallel-jobs=4'
+# install last released miopentensile in default, install latest commits when MIOTENSILE_VER="latest"
+RUN if [ "$MIOTENSILE_VER" = "latest" ] ; then cget -p $PREFIX install -DAMDGPU_TARGETS=${GPU_ARCH} ROCmSoftwarePlatform/MIOpenTensile@e258bbaed0bd9de546ea38c9a5c42f71fa41d9a0; else cget -p $PREFIX install -DAMDGPU_TARGETS=${GPU_ARCH} ROCmSoftwarePlatform/MIOpenTensile@2e3e792b2674bf8cdf2620749a298ed5313351bb; fi
```

Comment:
`-DAMDGPU_TARGETS=${GPU_ARCH}` should be passed `cget init`. Also this should install tensile from a file so that hcc and hip-clang install the same version.

Reply:
Will move gpu_arch to init.
Currently both hcc and hip-clang are using the same version. The version divergence here is both for hip-clang. It intends to test both a stable version and a latest version of yaml+tensile.

Reply:
moved gpu_arch flag to cget init